.gitignore

```diff
@@ -30,5 +30,5 @@ src/svtk/svtk/utils/helpers.cpython-37m-darwin.so
 src/svtk/svtk/utils/s3bam.pyc
 src/svtk/svtk/utils/utils.pyc
 src/svtk/svtk/vcfcluster.pyc
-test/
-inputs/
+/test/
```

Comment:
Should this now be /test_inputs_*?

Reply:
Yes although following your suggestion below to consolidate the test input files under one directory I moved this to `/test_inputs`.